ivy/data_classes/container/experimental/layers.py

```diff
@@ -2280,3 +2280,112 @@ def rfftn(
             norm=norm,
             out=out,
         )
+
+    @staticmethod
+    def static_max_unpool1d(
+        input: ivy.Container,
+        indices: ivy.Container,
+        kernel_size: Union[Tuple[int], int],
+        strides: Union[int, Tuple[int]] = None,
+        padding: Union[int, Tuple[int]] = 0,
+        data_format: Union[str, ivy.Container] = "NWC",
+        /,
+        *,
```

Comment:
Misplaced `/, *` here (sorry to nitpick 😅)

Reply:
okay no problem.